spec/index_spec.js

```diff
@@ -0,0 +1,58 @@
+var index = require('../index.js');
+
+describe('index', function () {
+  describe('reponse message', function () {
+    beforeEach(function() {
+      process.env.REGEX_PULL_REQ_TITLE = '\\S\\s';
+      process.env.REGEX_PULL_REQ_BODY = '\\S\\s';
+      process.env.ENABLE_KEYWORD_CHECKER = 'false';
+      process.env.CONTRIBUTING_GUIDELINES = 'test';
+    });
+
+    afterEach(function() {
```

Comment:
Since you are resetting the env variables on `beforeEach`, I think this is redundant.